test/contrib/rack/custom/distributed_custom_headers_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'contrib/rack/helpers'
+require 'contrib/http/test_helper'
+
+# Below a quick example on how to hack and modify the standard headers.
```

Comment:
I'm not really convinced that we want to provide such kind of suggestion (and your comment already suggest to not change headers). Distributed Tracing wasn't part of previous releases (at least not in the sense of stable and fully supported API), and writing this example here means suggesting to monkey patch our library to change headers name.

This should never happen because it means providing the same possibility to all other libraries (e.g. Ruby net/http with custom headers -> Django web framework). We may suggest this approach in an informal way, but I'm not sure to write it here, since it's like "you may do this but don't!".

I'd advocate to support that functionality 100% or 0%. Providing a suggestion in a GitHub comment and go back to the problem if it's happening very often, may be enough.